### DISCUSSION TITLE
Order from fields not right.

### DISCUSSION THREAD

**Discussion by harmkroog** [Budibase/budibase#10503] [Help] [answered]
Hi, i dont understand but in Data view the order is right.
Fields from left to right
aktiv , rechnungsmeldeadresse, etc.

But in GUI the order is total random and buttons are at the end.
if i moved fields in the database, button shown in the middle.
with the button configure columns the oder ist not the order it should.
here schreeshot what happens.
the order is mirrored and mixed

![Auswahl_114](https://user-images.githubusercontent.com/130768693/236635136-1a5ad994-d6a2-41d2-bccd-099bf51214ae.png)
![Auswahl_113](https://user-images.githubusercontent.com/130768693/236635141-b5ea0003-e49b-40fe-8662-094060bb76d9.png)

If i change the order in the Dataprovider.
eg aktiv is at the last in the view but in column config at first.
if i push at at the last. it will show in the middle

![Auswahl_115](https://user-images.githubusercontent.com/130768693/236636525-17d5f483-bc96-4d47-9d93-81dc172b1057.png)

![Auswahl_116](https://user-images.githubusercontent.com/130768693/236636539-727a1dac-6cb3-46b1-a4fd-9f9065923d1b.png)

I hope anyone can help me.
Thank you!
best regards Bonkersdeluxe

**Comment by ConorWebb96** [comment]
Hey @harmkroog,

Can you update to the latest version 2.6.8 and see if this resolves the issue. I haven't been able to recreate the issue you are experiencing on this version.

Thanks in advance.

**Comment by harmkroog** [comment]
Thank you!
But i use the docker image and there is over docker pull now new release yet.
I must wait or can i update it on an other way?
Thank you!
Best regards Bonkersdeluxe

**Comment by melohagan** [comment, marked as answer]
Hey @harmkroog I have a PR out that should hopefully resolve this once merged: https://github.com/Budibase/budibase/pull/10627
